sospice/util/sigma_clipping.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from scipy.ndimage import generic_filter
+from numpy import ma
+
+
+def _stddev_loc(image, size):
+    if type(size) is int:
+        size = (size,) * image.ndim
+    mean2 = _blur(image, size) ** 2
+    vari = _blur(image ** 2, size)
+    vari -= mean2
+    vari[vari <= 0] = 1e-20
```

Comment:
I guess this is only because of rounding errors giving small negative values.

Wouldn't setting the < 0 values to 0 be sufficient?